imod_coupler/kernelwrappers/msw_wrapper.py

```diff
@@ -0,0 +1,177 @@
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+         NDArray[np.float_]:
+            surface water timestep length in days
+        """
+        dtsw = self.get_value("dtsw")
+        return float(dtsw[0])
+
+    def get_surfacewater_sprinking_demand_ptr(self) -> NDArray[np.float_]:
+        """returns the sprinkling volume demand from metaswap
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+         NDArray[np.float_]:
+            sprinkling demand of MetaSWAP in m3/ dtgw. Array as pointer of the MetaSWAP intenal array.
+            Internally MetaSWAP uses a different array for get and set operations.
+        """
+        return self.get_value_ptr("ts2dfmputsp")
+
+    def get_surfacewater_sprinking_realised_ptr(self) -> NDArray[np.float_]:
+        """sets the sprinkling volume demand in metaswap.
+
+        Parameters
+        ----------
+        sprinkiling_demand: NDArray[np.float_]:
+            sprinkling demand of MetaSWAP in m3/dtgw
+
+        Returns
+        -------
+        none
+
+        """
+        return self.get_value_ptr("dfm2tsgetsp")
+
+    def get_surfacewater_ponding_allocation_ptr(self) -> NDArray[np.float_]:
+        """retruns the pointer to the ponding volume allocation array in MetaSWAP.
+        MetaSWAP uses two different ponding volume arrays. One for ponding allocation at the beginning of a (sub) timestep
+        and one for the returned realised volume at the end of the (sub) timestep.
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+         NDArray[np.float_]:
+            ponding volume allocation of MetaSWAP in m3/dtsw. Array as pointer of the MetaSWAP intenal array.
+            Internally MetaSWAP uses a different array for get and set operations.
+        """
+        return self.get_value_ptr("ts2dfmput")
+
+    def get_surfacewater_ponding_realised_ptr(self) -> NDArray[np.float_]:
+        """retruns the pointer to the ponding volume realised array in metaSWAP
+        MetaSWAP uses two different ponding volume arrays. One for ponding allocation at the beginning of a (sub) timestep
+        and one for the returned realised volume at the end of the (sub) timestep.
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+        none
+        """
+        return self.get_value_ptr("ts2dfmget")
+
+    def get_ponding_level_2d_ptr(self) -> NDArray[np.float_]:
+        """get ponding level from dlfow-2d in metaswap
+
+        Parameters
+        ----------
+        none
+
+        Returns
+        -------
+        ponding_level_2d: NDArray[np.float_]
+            ponding level 2d
+        """
+
+        return self.get_value_ptr("dfm2lvswk")
+
+    def get_svat_area_ptr(self) -> NDArray[np.float_]:
+        """gets area's of svats in metaswap. This can ben used to calculate ponding volumes based on dlfow ponding levels
```

Comment:
```suggestion
        """
        Gets areas of svats in metaswap. This can ben used to calculate ponding volumes based on dflow ponding levels
```